diff --git a/node_modules/@alita/plugins/templates/keepalive/context.tpl b/node_modules/@alita/plugins/templates/keepalive/context.tpl
index 51fc022..e117bfd 100644
--- a/node_modules/@alita/plugins/templates/keepalive/context.tpl
+++ b/node_modules/@alita/plugins/templates/keepalive/context.tpl
@@ -25,8 +25,13 @@ import { useIntl } from '../exports';
 const { TabPane } = Tabs;
 {{/hasTabsLayout}}
 {{/hasCustomTabs}}
+import dayjs from 'dayjs';
 
 
+const formatDate = (date: string | Date | number, format = 'YYYY-MM-DD') => {
+  return dayjs(date).format(format);
+};
+
 
 export const KeepAliveContext = React.createContext({});
 // 兼容非全路径的 path
@@ -320,8 +325,22 @@ export function useKeepOutlets() {
 {{/hasCustomTabs}}
         {
             Object.entries(keepElements.current).map(([pathname, { children }]: any) => (
-                <div key={`${pathname}:${cacheKeyMap[pathname] || '_'}`} style={ { height: '100%', width: '100%', position: 'relative', overflow: 'hidden auto' } } className="runtime-keep-alive-layout" hidden={!matchPath(location.pathname, pathname)}>
-                    {children}
+                <div key={`${pathname}:${cacheKeyMap[pathname] || '_'}`} style={ { height: '100%', width: '100%', position: 'relative', overflow: 'hidden auto',display: 'flex', flexDirection: 'column' } } className="runtime-keep-alive-layout" hidden={!matchPath(location.pathname, pathname)}>
+                    <div style={ { flex: 1 } }>
+                      {children}
+                    </div>
+                    <div style={ { position: 'relative',width:'100%',height:'80px',display:'flex',justifyContent: 'center',zIndex: 2 } }>
+                      <span>
+                        杭州雷数科技有限公司 版权所有 ©️ 2017-{formatDate(new Date(), 'YYYY')} All Rights Reserved
+                      </span>
+                      <a
+                        href="http://beian.miit.gov.cn"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        浙ICP备17054225号-1
+                      </a>
+                    </div>
                 </div>
             ))
         }
